### DISCUSSION TITLE
disable static bucket

### DISCUSSION THREAD

**Discussion by atomraymaker** [architect/architect#1321] [Q&A] [answered]
I don't need static assets right now and I don't want to fight with the account holders about allow a public bucket, is there a way to disable the static bucket creation when using @http?

**Comment by atomraymaker** [comment]
or maybe there is a way to add cloudfront in front of the bucket so that the bucket can be private? I can't see either of these options on the docs so far.

**Comment by ryanblock** [comment, marked as answer]
I’d suggest just removing it from the cloudformation template with a deploy.start plugin!
